add --time-style option in long style format

t-mangoe opened the pull request:
### Description

Thanks for contributing this Pull Request. Add a brief description of what this Pull Request does. Do tag the relevant issue(s) and PR(s) below. If required, add some screenshot(s) to support your changes.

I added `--time-style` option.
This PR closes #395 issues.

- Relevant Issues : #395 
- Relevant PRs : (none)
- Type of change :
  - [x] New feature
  - [ ] Bug fix for existing feature
  - [ ] Code quality improvement
  - [ ] Addition or Improvement of tests
  - [ ] Addition or Improvement of documentation

t-mangoe commented:
If you don't mind, could I write my GitHub account name in `CONTRIBUTORS.md` ?

t-mangoe commented:
Thank you.
I wrote my GitHub account name in `CONTRIBUTORS.md`.

athityakumar commented:
@t-mangoe - Can you also add this new flag in the `.travis.yml` as well as `tab_complete.sh`?

avdv commented:
> Can you also add this new flag in the `.travis.yml` as well as `tab_complete.sh`?

@athityakumar Please note that Travis CI (org) has been shut down. I asked you to migrate to Travis-ci.com a bit over a year ago, but that never happened. So, I removed the travis integration in commit c0459f71bd892598eadbad38dd72614915318409.

We are using Github Actions now. :cat: (see #458)  But that only runs a simple `rake` command which does not ensure the tab completion files are up-to-date.

athityakumar commented:
@avdv - Sorry, my bad! Thanks for refreshing my memory 😄 

We can add the rest of the `colorls` commands which were present as `script`s in .travis.yml to the `ruby.yml` file too right? GH Actions is pretty much becoming the standard for CI - so, let's add the rubygems publishing as well as colorls flag commands as part of GH Actions workflow itself. We can create a new GH issue for this though.

@t-mangoe - You can add the new flag just to `tab_complete.sh` as part of this PR, and we should be good to merge 😄

avdv reviewed (changes requested):
@t-mangoe  Please make the --time-style option compatible to GNU ls, like we had agreed:

>  --time-style=TIME_STYLE
>              time/date format with -l; see TIME_STYLE below

> The TIME_STYLE argument can be full-iso, long-iso, iso, locale, or +FORMAT.  FORMAT is interpreted like in date(1).

avdv commented:
>  You can add the new flag just to `tab_complete.sh` as part of this PR

No need to touch that file. The completions for Bash are always up-to-date, since they are requested dynamically from colorls itself.

For zsh, it's just a matter of running `rake zsh/_colorls` and commiting the changed file.

t-mangoe commented:
@avdv
Thank you for your review!!
I changed the option's format in '+FORMAT'. To be accurate, I changed the format so that the '+' character is required at the beginning.

avdv reviewed (approved):
Thank you, @t-mangoe !

fharper commented:
@t-mangoe : thanks a lot for that PR 🎉
